newspapers.com.js

```diff
@@ -84,14 +85,13 @@ function doWeb(doc, _url) {
 
 	newItem.abstractNote = details.media.note;
 	
-	/*
-	<meta property="og:image" content="https://img0.newspapers.com/img/img?id=97710064&width=557&height=4616&crop=1150_215_589_4971&rotation=0&brightness=0&contrast=0&invert=0&ts=1467779959&h=e478152fd53dd7afc4e72a18c1dad4ea">
-	*/
-	newItem.attachments = [{
-		url: metaArr["og:image"],
-		title: "Image",
-		mimeType: "image/jpeg"
-	}];
+	var uniqueID = newItem.url.match(/\d+/);
```

Comment:
could you make this a bit stricter, e.g. `(/\/clip\/(\d+)/)[1]` your version would fail if e.g. the article title contains a year

Reply:
the site removes slashes from the download url - here's a test example, where the clip title was "test 1969/01/01"

https://www.newspapers.com/clip/62062519/test-19690101/ 

download is working on my test run

Reply:
I'm not worried about the slashes, I'm worried about the fact that your match would return an array with multiple items (run `Z.debug(uniqueID)`  in that example to see) and I'd rather not rely on whatever array to string forcing javascript does to get that right. Better to match the right string (i.e. the numbers after `clip/` always.

Reply:
I see. I'll fix it up and resubmit. Any advice on the failed check? The job log references a high vulnerability but I'm not catching what it's trying to tell me.

Reply:
Don't worry about the failed checks — this branch is old, and the checking code is outdated.

(If you know how to rebase on the current master, you can do that and force push, and that might fix the checks.)

Reply:
I don’t know how to do that but I could look into it this weekend. Will it
be necessary to do so, to finish this up? Thanks!

Reply:
nah -- I can just run a check locally, don't worry about rebasing